deployment/k8s/charts/aggregates/testClusters/environments/es-2-4-single-node-cluster.yaml

```diff
@@ -0,0 +1,36 @@
+conditionalPackageInstalls:
+  proxy: false
+  source: true
+  target: false
+
+
+# Elastic docker registry does not seem to include 2.x versions and thus being pulled from docker hub
+source:
+  imageTag: "2.4.5"
+  image: "elasticsearch"
+  antiAffinity: "soft"
+  esJavaOpts: "-Xmx128m -Xms128m"
+  protocol: http
+  replicas: 1  # Set replicas to the number of nodes you want in the cluster (e.g., 3)
+  createCert: false
+  sysctlInitContainer:
+    enabled: false
+  esConfig:
+    elasticsearch.yml: |
+      bootstrap.system_call_filter: false
+      xpack.security.enabled: false
+      network.host: 0.0.0.0
+  extraEnvs:
+    - name: "cluster.initial_master_nodes" # Remove setting unknown to ES 2.x
+      value: ""
```

Comment:
by "remove", this means set to empty string? (basically, is this a todo comment or this is the implementation?)